app/src/main/java/com/sopt/now/compose/ui/screens/login/LoginViewModel.kt

```diff
@@ -12,38 +13,35 @@ import androidx.navigation.NavHostController
 import com.sopt.now.compose.MainActivity.Companion.NAVIGATE_BACK_PRESSED_KEY
 import com.sopt.now.compose.MainActivity.Companion.NAVIGATE_LOGIN_KEY
 import com.sopt.now.compose.MainActivity.Companion.NAVIGATE_SIGNUP_KEY
-import com.sopt.now.compose.R
 import com.sopt.now.compose.SoptApplication
 import com.sopt.now.compose.container.PreferenceUserRepository
 import com.sopt.now.compose.models.User
+import com.sopt.now.compose.network.ServicePool.authService
+import com.sopt.now.compose.network.dto.RequestLoginDto
+import com.sopt.now.compose.network.dto.ResponseLoginDto
 import com.sopt.now.compose.ui.navigation.HomeDestination
 import com.sopt.now.compose.ui.navigation.SignUpDestination
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.jsonObject
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
-class LoginViewModel(
-    val userRepository: PreferenceUserRepository
-): ViewModel() {
+class LoginViewModel(): ViewModel() {
```

Comment:
```suggestion
class LoginViewModel: ViewModel() {
```

생성자에 아무것도 안들어가면 괄호 지워도 됩니다.